packages/backend/src/execution/executors/PgExecutor/PgExecutor.test.ts

```diff
@@ -1,26 +1,28 @@
-import { col } from '@synthql/queries';
+import { col, query } from '@synthql/queries';
 import { describe, expect, it } from 'vitest';
 import { PgExecutor } from '.';
-import { from } from '../../../tests/generated.schema';
+import { DB } from '../../../tests/generated';
 import { pool } from '../../../tests/queryEngine';
 import { createRefContext } from '../../../refs/RefContext';
 import { QueryProviderExecutor } from '../QueryProviderExecutor';
 import { collectColumnReferences } from '../../../query/collectColumnReferences';
 
+const from = query<DB>().from;
```

Comment:
You should modify the generator logic to append this to the /introspection index generation logic so you don't have to keep re-typing this everywhere. Something like:

```ts
fs.writeFileSync('index.ts',`const from = query<DB>().from;`)


```